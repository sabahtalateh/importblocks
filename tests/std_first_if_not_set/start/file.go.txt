package test1

import (
	"x/y/z"
	"a/b/c"
	"os"
	"a/b"
)
