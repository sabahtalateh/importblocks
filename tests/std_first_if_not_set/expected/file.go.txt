package test1

import (
	"os"

	"a/b"
	"a/b/c"

	"x/y/z"
)
